lambda/src/main/scala/pricemigrationengine/migrations/GuardianWeekly2025Migration.scala

```diff
@@ -207,7 +207,7 @@ object GuardianWeekly2025Migration {
 
     val order_opt = {
       for {
-        ratePlan <- SI2025RateplanFromSubAndInvoices.determineRatePlan(subscription, invoiceList)
+        ratePlan <- SI2025RateplanFromSubAndInvoices.determineRatePlan(zuora_subscription, invoiceList)
```

Comment:
`val zuora_subscription = subscription` is a curious line - why not rename the function parameter?